src/plugins/formvalid/formvalid.js

```diff
@@ -319,8 +319,16 @@ var componentName = "wb-frmvld",
 											// Create our container if one doesn't already exist
 											if ( $summaryContainer.length === 0 ) {
 												$summaryContainer = $( "<section id='" + errorFormId + "' class='alert alert-danger' tabindex='-1'>" + summary + "</section>" ).prependTo( $form );
-											} else {
+											} else if ( $summaryContainer.html().replace( /'/g, "\"" ).replace( /&#160;/g, "&nbsp;" ) !== summary.replace( /'/g, "\"" ).replace( /&#160;/g, "&nbsp;" ) ) {
```

Comment:
@radmorecameron I agree that the solution doesn't seem the best. However, I have tried to find a better solution on my end, but couldn't come up with one. That said, it doesn't seem necessary to do the replace on `$summaryContainer.html()`. We use single quotes in `summary` because of our code normalization and the `&#160;` is there because of the resource files.

Also, would you please add a comment to explain why the `else if` is there? See next comment